package.json

```diff
@@ -1,27 +1,71 @@
 {
-  "dependencies": {
-    "request": "2.88.0",
-    "q": "0.9.4"
-  },
-  "readmeFilename": "README.md",
-  "gitHead": "0faea06f81f0b3c533ec569c5e8f25a9152f444c",
   "name": "jotform",
-  "description": "jotform API nodejs client",
-  "version": "0.1.5",
-  "directories": {
-    "test": "test"
+  "version": "0.2.0",
+  "description": "Jotform API Node.js Client",
+  "type": "module",
+  "sideEffects": false,
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/esm/index.js",
+  "source": "./src/index.ts",
+  "types": "./dist/cjs/index.d.ts",
+  "exports": {
+    "import": "./dist/esm/index.js",
+    "require": "./dist/cjs/index.js"
   },
-  "devDependencies": {},
   "scripts": {
-    "test": "mocha test/*.js"
+    "build": "yarn build-esm && yarn build-cjs && yarn build-cjs-package",
+    "build-esm": "tsc --project tsconfig.build.json --outDir dist/esm --module esnext",
+    "build-cjs": "tsc --project tsconfig.build.json --outDir dist/cjs --module commonjs",
+    "build-cjs-package": "echo '{\n  \"type\": \"commonjs\"\n}' > dist/cjs/package.json",
+    "clean": "rimraf dist",
+    "lint": "eslint .",
+    "postinstall": "husky install",
```

Comment:
This is incredible thank you for doing this! 
I am running into a `husky install` issue when I try to install your fork in my project.

```
> api@0.0.1 prepare
> husky install

husky - .git can't be found (see https://typicode.github.io/husky/#/?id=custom-directory)
npm ERR! code 1
npm ERR! path /Users/yvesikar/Documents/portfolio/fertility/fertility-co/api
npm ERR! command failed
npm ERR! command sh -c husky install

npm ERR! A complete log of this run can be found in: /Users/yvesikar/.npm/_logs/2023-09-01T23_34_43_714Z-debug-0.log
```

I think this is because of the [postinstall script](https://github.com/wojtekmaj/jotform/blob/f44bf07614bcdcd70bdddc1d426bc2d945247363/package.json#L22C1-L23C1) because removing it on my own fork installs without error.

Reply:
Hmm what worries me here is that I see error messages from npm - while I opted for Yarn 3. I don't think it's critical to use Yarn if you prefer npm - I can happily migrate if you'd like me to.

Thanks for the kind words!

Reply:
Yes just to clarify I am installing the package via `npm@9.6.7`.

I am not that familiar with how husky works but I see [this file](https://github.com/wojtekmaj/jotform/blob/f44bf07614bcdcd70bdddc1d426bc2d945247363/.husky/pre-commit#L2C1-L2C1) is referring to a husky.sh file that doesn't exist in the repo?